### ISSUE TITLE
Баг автосалона грузовиков.

### ISSUE THREAD

**Issue by AppleJucke** [LambdaRP/Bug-Tracker-LambdaRP#32]
Во вкладке прицепов для тягачей, зависает при прокликивании на первой же позиции.
Выглядит собственно как пофикшенный баг в автосалоне, но проявляется только на прицепах.

**Comment by KatzzzGG** [comment]
Да, очень бы хотелось побыстрее получить доступ к возможности покупки прицепов...
